server/app/uk/gov/ons/addressIndex/server/controllers/PlayHelperController.scala

```diff
@@ -0,0 +1,124 @@
+package uk.gov.ons.addressIndex.server.controllers
+
+import play.api.libs.json.{Json, Writes}
+import play.api.mvc.{Controller, Result}
+import scala.concurrent.Future
+
+trait PlayHelperController extends Controller {
+
+  /**
+    * Use this instead of Future.successful(BadRequest(Json.toJson(toJsonable)))
+    *
+    * @param toJsonable an object which has an implicit Writes (PlayJson)
+    * @param writes implicit Writes
+    * @tparam T type of the toJsonable
+    * @return a Future BadRequest with a Json body
+    */
+  def futureJsonBadRequest[T](toJsonable: T)(implicit writes: Writes[T]): Future[Result] = {
```

Comment:
The idea is that you use this helper, with a descriptive name.  Please see scaladoc.